doc/jemalloc.xml.in

```diff
@@ -2623,8 +2629,8 @@ struct extent_hooks_s {
           <literal>r-</literal>
           [<option>--enable-stats</option>]
         </term>
-        <listitem><para>Cumulative number of small allocation requests.
-        </para></listitem>
+        <listitem><para>Cumulative number of allocation requests satisfied by
+        all bin size classes.</para></listitem>
```

Comment:
I was confused for a bit while updating these docs about the distinction between "bin" and "small", and realized that ideally we wouldn't talk about "bins" at all and call them [slab-backed] small size classes (probably also renaming "hbins" to something else).  But, that's a non-trivial renaming that I'm reluctant to do so close to the release.